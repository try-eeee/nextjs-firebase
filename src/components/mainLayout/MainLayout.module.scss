.wrapper {
  padding: 6rem;
}
