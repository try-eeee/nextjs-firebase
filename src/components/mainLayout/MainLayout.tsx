import styles from "./MainLayout.module.scss";

interface MainLayoutProps {
  children: React.ReactNode;
}

export function MainLayout(props: MainLayoutProps) {
  const { children } = props;
  return <div className={styles.wrapper}>{children}</div>;
}
