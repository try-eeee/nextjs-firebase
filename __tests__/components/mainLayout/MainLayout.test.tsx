import React from "react";
import { render } from "@testing-library/react";
import { MainLayout } from "@/components/mainLayout/MainLayout";

describe("MainLayout UnitTest", () => {
  test("Snapshot Test Of MainLayout", () => {
    const { asFragment } = render(
      <MainLayout>
        <>コンテンツ</>
      </MainLayout>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
