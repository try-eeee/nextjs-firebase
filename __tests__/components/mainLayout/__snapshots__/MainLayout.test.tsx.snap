// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout UnitTest Snapshot Test Of MainLayout 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    コンテンツ
  </div>
</DocumentFragment>
`;
